packages/react-components/react-swatch-picker-preview/src/components/ColorSwatch/ColorSwatch.test.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { isConformant } from '../../testing/isConformant';
+import { ColorSwatch } from './ColorSwatch';
+import { colorSwatchClassNames } from './useColorSwatchStyles.styles';
+
+describe('ColorSwatch', () => {
+  isConformant({
+    Component: ColorSwatch,
+    displayName: 'ColorSwatch',
+    testOptions: {
+      'has-static-classnames': [
+        {
+          props: {},
+          expectedClassNames: {
+            root: colorSwatchClassNames.root,
+          },
+        },
+      ],
+    },
+  });
+
+  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
```

Comment:
```suggestion
```

not sure why add comments about vr-tests addition to jest tests

if you wanna be explicit about adding more jest tests lets use `it.todo('...more tests will be added')`

Reply:
in general a pattern to follow - always make tests part of your implementation logic. otherwise how can we verify that the implementation works

Reply:
Those comments were generated. But to add tests at once is a good idea

Reply:
removed those TODOs as visual regression will be added later. Currently visuals are not implemented yet